app/models/comment.rb

```diff
@@ -0,0 +1,5 @@
+class Comment < ApplicationRecord
+  belongs_to :review
+  belongs_to :user
+  validates :comment, presence: true
```

Comment:
@ard-wtag I think we should add an empty line here.

Reply:
added bhai